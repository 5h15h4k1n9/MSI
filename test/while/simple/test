countUpTo(5)
countUpTo(0)
sumTo(3)
sumTo(0)
doubleUntil(10)
doubleUntil(1)
exit
