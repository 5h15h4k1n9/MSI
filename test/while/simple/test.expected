(int) 5
(int) 0
(int) 6
(int) 0
(int) 16
(int) 1
Bye-bye!
